controller_manager:
  ros__parameters:
    update_rate: 50

diff_drive_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: "bl_wheel_joint"
  right_wheel: "br_wheel_joint"
  pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000000.0]
  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000000.0]

  wheel_separation: 0.27
  wheel_radius: 0.03

  cmd_vel_timeout: 0.5
  publish_rate: 50